crates/sui-framework/packages/sui-framework/sources/display_registry.move

```diff
@@ -0,0 +1,95 @@
+module sui::display_registry;
+
+use std::string::String;
+use sui::display::{Self, Display};
+use sui::dynamic_field;
+use sui::package::Publisher;
+use sui::transfer::Receiving;
+use sui::vec_map::VecMap;
+
+const EUnauthorized: u64 = 0;
+const EDisplayAlreadyExists: u64 = 1;
+const ENotSystemAddress: u64 = 2;
+
+public struct Registry has key { id: UID }
+public struct DisplayKey<phantom T>() has copy, drop, store;
+public struct DisplayCap<phantom T> has key, store { id: UID }
+public struct DisplayV2(VecMap<String, String>) has copy, drop, store;
```

Comment:
If this type is not accessible outside, you probably don't need the type wrapper.